src/util/utils.ts

```diff
@@ -60,3 +60,52 @@ export const formatImageURL = (
   return `${url}.${format ||
     (url.includes("/a_") ? "gif" : "jpg")}?size=${size}`;
 };
+
+export function camelToSnakeCase(str: string) {
+  return str.replace(/ID|[A-Z]/g, (s) => {
+    if (s === "ID") return "_id";
+    return `_${s.toLowerCase()}`;
+  });
+}
+
+export function snakeToCamelCase(s: string) {
+  return s.replace(/_id|([-_][a-z])/ig, ($1) => {
+    if ($1 === "_id") return "ID";
+    return $1.toUpperCase().replace("_", "");
+  });
+}
+
+export function isObject(o: unknown) {
+  return o === Object(o) && !Array.isArray(o) && typeof o !== "function";
+}
+// deno-lint-ignore no-explicit-any
+export function camelKeysToSnakeCase(o: any) {
```

Comment:
```suggestion
export function camelKeysToSnakeCase(o: Record<string, any>) {
```
Also, a better parameter name like `obj`.